scripts/monitor/index.js

```diff
@@ -17,7 +17,10 @@ Promise.resolve()
   })
   .then(() => {
     log.debug('Connecting to Ethereum node')
-    return eth.connect()
+    return eth.connect({
```

Comment:
eth (or commons for that matter) doesn't relay on the ENV so if you want to use the RPC_URL you have to explicity use it like you did here, that's why it wasn't working.

I think the contracts can be left out because the EventMonitor sets them dynamically here https://github.com/decentraland/marketplace/blob/master/scripts/monitor/EventMonitor.js#L67